src/test/ui/traits/wf-trait-object-reverse-order.rs

```diff
@@ -0,0 +1,15 @@
+// run-pass
+
+// Ensure that `dyn $($AutoTrait) + ObjSafe` is well-formed.
+
+use std::marker::Unpin;
+
+// Some arbitray object-safe trait:
+trait Obj {}
+
+type _0 = Unpin;
+type _1 = Send + Obj;
```

Comment:
is there a reason not to have the `dyn` keyword here?

Reply:
Just a mistake... :)